src/main/java/org/walkmod/WalkModFacade.java

```diff
@@ -219,6 +223,58 @@ public WalkModFacade() {
 		this(false, true, false);
 	}
 
+
+	/**
+	 * Initalizes a Walkmod service
+	 *
+	 * @param walkmodCfg
+	 *            Walkmod configuration file. If null, file named 'walkmod.xml'
+	 *            is searched at the root.
+	 * @param options
+	 *            Map of option. See {@link Options} and {@link OptionsBuilder}
+	 *            for available options and default values.
+	 * @param configurationProvider
+	 *            Configuration provider responsible for the resolution of
+	 *            plugins (used to use custom classloading strategies).
+	 *            If null Ivy is used.
+	 */
+	public WalkModFacade(File walkmodCfg, Map<String,Object> options, ConfigurationProvider configurationProvider) {

```

Comment:
What I mean is that I agree with using OptionsBuilder. We have a pletora of overloaded methods in [Asciidoctor]( https://github.com/asciidoctor/asciidoctorj/blob/master/asciidoctorj-core/src/main/java/org/asciidoctor/Asciidoctor.java#L43,L73) and from time to time someone asks about the available options.
Using only OptionBuilder, make things clearer.